src/sentry/api/endpoints/auth_index.py

```diff
@@ -94,6 +95,7 @@ def _verify_user_via_inputs(validator, request):
                 )
         # attempt password authentication
         elif "password" in validator.validated_data:
+            metrics.incr("password.login_attempt", sample_rate=1.0, skip_internal=False)
```

Comment:
Same as above. Would it be better to put it below so we can be sure that people are successfully logging in using password?

The current implementation means that if Line 99 `check_password` function breaks, it would never surface on our dashboards.

Reply:
Fixed in https://github.com/getsentry/sentry/pull/47998